tex/chProofs.tex

```diff
@@ -1702,6 +1788,8 @@ \subsection{Proofs by induction}\label{ssec:ind}
 instance the induction principle for sequences, which has some
 similarities with the one for natural numbers:
 
+% [DG] Looks weird to summon the list type, after having worked with
```

Comment:
In case you're curious: `seq` is defined [here](https://math-comp.github.io/math-comp/htmldoc/mathcomp.ssreflect.seq.html#seq) as a notation for the `list` type:

```Coq
Notation seq := list.
```
Coq automatically generates induction principles for user-defined inductive types, hence the induction principle for `seq` is (perhaps surprisingly) named `list_ind`.